examples/transformers/whisper.py

```diff
@@ -1,8 +1,11 @@
-import base64
-import mlflow
 import transformers
 import requests
 
+import mlflow
+from mlflow.utils._cache_utils import _clear_hub_cache
+
+# Used only for CI testing - disregard for examples purposes
+_clear_hub_cache()
```

Comment:
Can we run this function in `test_examples.py`?

Reply:
Great idea :D